src/routes/balances/__tests__/controllers/get-externalized-balances.balances.controller.spec.ts

```diff
@@ -22,12 +22,24 @@ import { TestLoggingModule } from '@/logging/__tests__/test.logging.module';
 import { RequestScopedLoggingModule } from '@/logging/logging.module';
 import { NULL_ADDRESS } from '@/routes/common/constants';
 import { NetworkResponseError } from '@/datasources/network/entities/network.error.entity';
+import {
+  zerionAttributesBuilder,
+  zerionFungibleInfoBuilder,
+  zerionImplementationBuilder,
+  zerionQuantityBuilder,
+  zerionBalanceBuilder,
+  zerionFlagsBuilder,
+  zerionBalancesBuilder,
+} from '@/datasources/balances-api/entities/__tests__/zerion-balance.entity.builder';
 
 describe('Balances Controller (Unit)', () => {
```

Comment:
For readability, I'd say to split the controller tests between the two providers. What do you think?

Reply:
Yes, I think it makes sense, thanks! 

Done in [4d522b5](https://github.com/safe-global/safe-client-gateway/pull/1096/commits/4d522b531cbc41a12aab02d271404db7e1cb5a96)